src/osdc/Striper.cc

```diff
@@ -407,7 +407,7 @@ void Striper::StripedReadResult::add_partial_result(
 		 << " to " << buffer_extents << dendl;
   for (auto& be : buffer_extents) {
     auto& r = partial[be.first];
-    size_t actual = std::min<uint64_t>(bl.length(), be.second);
+    size_t actual = std::min<uint64_t>(bl.length(), be.second); //NOLINT(bugprone-use-after-move)
```

Comment:
Please add (to the commit message) an explanation of why this is a false positive.
Remember that it took us a few minutes to understand the reason when discussing this. The reader
should not be forced to go thru the same deliberations.

Reply:
@ronen-fr I have added an explanation in the commit as well as in the PR